uberfire-workbench/uberfire-workbench-client-views-patternfly/src/main/resources/org/uberfire/client/views/PatternFly.gwt.xml

```diff
@@ -30,4 +30,5 @@
   <stylesheet src="bootstrap-select/css/bootstrap-select.min.css"/>
   <stylesheet src="prettify/bin/prettify.min.css"/>
   <stylesheet src="uberfire-patternfly.css" />
+  <stylesheet src="monaco-editor/min/vs/editor/editor.main.css"/>
```

Comment:
Did you want to keep this as `dev` given your comment about using the `min` version?

Reply:
I think we're fine regarding CSS compression, Mike :-)